bundles/AdminBundle/src/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -373,7 +373,7 @@ public function doGetGridColumnConfig(Request $request, Config $config, bool $is
                 $request->get('no_system_columns', false),
                 $class,
                 $gridType,
-                $request->get('no_brick_columns', false),
+                $request->get('no_brick_columns', false) === 'true',
```

Comment:
please revert this change if [this ](https://github.com/pimcore/pimcore/pull/13981/files#diff-c091176bf3a4d76953d98f04487e90a4d4509d402621e8f588f2cdc215fa865bR374) gets merged.